beacon-chain/sync/pending_attestations_queue_test.go

```diff
@@ -256,6 +261,168 @@ func TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra(t *testing.T) {
 	cancel()
 }
 
+func TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra_VerifyAlreadySeen(t *testing.T) {
```

Comment:
added unit test from https://github.com/prysmaticlabs/prysm/pull/15029 verifying the cache